### PR TITLE
Added support for GraphQL

### PR DESCRIPTION
**Pull request by EricWittmann**
Basic ArtifactType support for GraphQL artifacts.  Canonicalization is implemented as well as the Validity rule.  Compatibility is not implemented.

### CODE REVIEW

File: app/src/main/java/io/apicurio/registry/content/canon/GraphQLContentCanonicalizer.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2019 Red Hat
```
Comment (alesj) [inline app/src/main/java/io/apicurio/registry/content/canon/GraphQLContentCanonicalizer.java]: 2020 :-)

Comment (EricWittmann) [reply app/src/main/java/io/apicurio/registry/content/canon/GraphQLContentCanonicalizer.java]: Damn you, copy/paste!! :)

Comment (EricWittmann) [reply app/src/main/java/io/apicurio/registry/content/canon/GraphQLContentCanonicalizer.java]: I realized I need a unit test for this anyway - so I'll add a unit test and fix the (c) year.

**Comment by alesj** [comment]
Looks OK.

**Pull request merged**